### ISSUE TITLE
Double Text labels, when none should be shown

### ISSUE THREAD

**Issue by TLoATDaE** [benbasson/imagetoolbar#21]
Lately, after I have set things to show no text labels, there are a few places/websites where it not only show the text labels, but stacks them! It make it very odd, as it takes up a lot of real-estate on the image. Have a look:
![double label example](https://cloud.githubusercontent.com/assets/18477582/14553203/9bd4eb84-02b2-11e6-9faf-dba0366daf18.jpg)

**Comment by yfdyh000** [comment]
I have seen this problem today, but without a reliable steps to reproduce.

**Comment by yfdyh000** [comment]
Testcase:
```
<style>
label{display:inline-block;margin-bottom:5px;font-weight:bold}
</style>
<p><img src="https://addons.cdn.mozilla.net/user-media/previews/full/2/2553.png" ></p>
```

The toolbar element is inserted directly into the pages and not isolated, it may be impact from style sheet on the pages.

**Comment by benbasson** [comment]
Will be fixed by #24
